governator-legacy/src/main/java/com/netflix/governator/guice/InternalAutoBindModuleBootstrapModule.java

```diff
@@ -63,6 +63,7 @@ private Module bindAutoBindSingletons(BootstrapBinder binder) {
                         "@AutoBindSingleton(multiple=true) value cannot be set for Modules");
 
                 LOG.info("Found @AutoBindSingleton annotated module : {} ", clazz.getName());
+                LOG.info("***** @AutoBindSingleton for module {} is deprecated as of 10/10/2015.  Modules should be added directly to the injector or via install({}.class). See https://github.com/Netflix/governator/wiki/Auto-Binding", clazz.getName(), clazz.getSimpleName());
```

Comment:
Guice4 is actually A LOT better at handing module dedup without having to add hashCode() and equals() module is doing something bad like bind(Foo.class).toInstance(new FooImpl()). 

The deprecation notice only shows up if the module is found via classpath scanning.  The warning will no longer be logged once users have switched to installing the module directly and @AutoBindSingleton is removed.